cargo/lib/dependabot/cargo/metadata_finder.rb

```diff
@@ -52,7 +52,6 @@ def crates_listing
 
         response = Excon.get(
           "https://crates.io/api/v1/crates/#{dependency.name}",
-          headers: { "User-Agent" => "Dependabot (dependabot.com)" },
```

Comment:
Should we also add the new headers here?

Reply:
The new header should get added along with `**SharedHelpers.excon_defaults` - wonder if cargo have any hard-coded rues matching on this? Will try and see where this came from.

Reply:
Ah yes ofc 👍